src/breakupModel/input/TLEReader.cpp

```diff
@@ -1,36 +1,69 @@
 #include "TLEReader.h"
 
+const std::map<char, size_t> TLEReader::alpha5NumberingSchemeOffset{
```

Comment:
you could also generate this map ;)

Reply:
Maybe yes...
But I would actually leave it as it is :)